src/interfaces/Movie.ts

```diff
@@ -0,0 +1,11 @@
+export interface Movie {
+  id: number;
+  title: string;
+  overview: string;
+  poster_path: string;
+  original_language: string;
+  genre_ids: [];
+  rate: number;
+  opinion: string;
+  movie_id: number;
+}
```

Comment:
Erre az interface-re nincs szükséged, mert már van a Movie-ból egy Model. A Model feltudod használni type-ként is, szóval az elég